probe-rs/src/rtt.rs

```diff
@@ -278,14 +278,14 @@ impl Rtt {
         }
 
         let cb_len = rtt_header.total_rtt_buffer_size();
+        let cb_content = ptr + rtt_header.header_size() as u64;
```

Comment:
Please do not shorten variable names :)

Reply:
It's also not the content, but the address of the content

Reply:
Just because I was asleep at the helm before does not mean I need to now :D

Reply:
IMHO since we have a bunch of `i`, `ptr`, `b` and other variables in this same file, pointing out _this one in particular_ feels a bit randomly nitpicky to me :)